core/ProxyHttp.php

```diff
@@ -170,6 +170,8 @@ public static function serverStaticFile($file, $contentType, $expireFarFutureDay
             Common::sendHeader('Content-Encoding: ' . $encoding);
         }
 
+        @ob_get_clean();
```

Comment:
As the buffer content does not matter and is not used, guess a `ob_end_clean()` might be better

Reply:
In WordPress the buffer may be used so didn't want to end them as wasn't sure if it could have any side effects. Shouldn't really make much of a difference? Although I suppose `ob_end_clean` would make sure it'll be printed I suppose? This is currently done in WordPress on shutdown by looping through all of them and then printing them basically. Eg ob_end_clean might call some callbacks that then will be executed and could break things when expecting some wrong output maybe not sure https://www.php.net/manual/en/function.ob-end-clean.php#42939

Reply:
According to https://www.php.net/manual/en/function.ob-get-clean.php `ob_get_clean` also executes `ob_end_clean`. So guess it doesn't make much difference...

Reply:
👍 didn't know. Changed it to ob_end_clean and also now iterating over multiple output bufferings